scripts/defaultScripts.js

```diff
@@ -71,23 +73,31 @@ if (Menu.menuExists(MENU_CATEGORY) && !Menu.menuItemExists(MENU_CATEGORY, MENU_I
         isChecked: previousSetting,
     });
 }
-
 function loadSeparateDefaults() {
     var currentlyRunningScripts = ScriptDiscoveryService.getRunning();
 
     for (var i in DEFAULT_SCRIPTS_SEPARATE) {
         var shouldLoadCurrentDefaultScript = true;
+        var scriptItem = DEFAULT_SCRIPTS_SEPARATE[i];
+        if (typeof scriptItem === "object") {
+            if (previousSettingBeta) {
+                console.info("Loading Beta item " + scriptItem.beta);
+                scriptItem = scriptItem.beta;
+            } else {
+                scriptItem = scriptItem.stable;
+            }
+        }
 
         for (var j = 0; j < currentlyRunningScripts.length; j++) {
             var currentRunningScriptObject = currentlyRunningScripts[j];
-            var currentDefaultScriptName = DEFAULT_SCRIPTS_SEPARATE[i].substr((DEFAULT_SCRIPTS_SEPARATE[i].lastIndexOf("/") + 1), DEFAULT_SCRIPTS_SEPARATE[i].length);
+            var currentDefaultScriptName = scriptItem.substr((scriptItem.lastIndexOf("/") + 1), scriptItem.length);
             if (currentDefaultScriptName === currentRunningScriptObject.name) {
-                shouldLoadCurrentDefaultScript = false;
+                ScriptDiscoveryService.stopScript(currentRunningScriptObject.url);
             }
         }
 
         if (shouldLoadCurrentDefaultScript) {
```

Comment:
`shouldLoadCurrentDefaultScript` is now always true, so can remove this `if()` and remove the variable declaration.